pkg/projects/descriptions.json

```diff
@@ -44,4 +44,9 @@
         "Description": "Provides a set of packages that can be used when building Universal Windows applications on .NETCore.",
         "CommonTypes": [ ]
     },
+    {
+        "Name": "Microsoft.NETCore.UniversalWindowsPlatform.CoreCLR",
+        "Description": "Provides a set of tools to run Universal Windows application on CoreCLR.",
```

Comment:
description can be improved 
this will provide the coreclr runtime appx package & files to build & run uwp apps using coreclr